src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -297,15 +285,40 @@ public Conference createConference(
         }
         while (conference == null);
 
+        return conference;
+    }
+
+    /**
+     * Initializes a new {@link Conference} instance with an ID unique to the
+     * <tt>Conference</tt> instances listed by this <tt>Videobridge</tt> and
+     * adds the new instance to the list of existing <tt>Conference</tt>
+     * instances. Optionally the new instance is owned by a specific conference
+     * focus i.e. further/future requests to manage the new instance must come
+     * from the specified <tt>focus</tt> or they will be ignored. If the focus
+     * is not specified this safety check is overridden.
+     *
+     * @param focus (optional) a <tt>String</tt> which specifies the JID of
+     * the conference focus which will own the new instance i.e. from whom
+     * further/future requests to manage the new instance must come or they will
+     * be ignored. Pass <tt>null</tt> to override this safety check.
+     * @param name world readable name of the conference to create.
+     * @param enableLogging whether logging should be enabled or disabled for
+     * the {@link Conference}.
+     * @param gid the optional "global" id of the conference.
+     * @return a new <tt>Conference</tt> instance with an ID unique to the
+     * <tt>Conference</tt> instances listed by this <tt>Videobridge</tt>
+     */
+    public Conference createConference(
+            Jid focus, Localpart name, boolean enableLogging, String gid)
+    {
+        final Conference conference = doCreateConference(focus, name, enableLogging, gid);
+
         // The method Videobridge.getConferenceCountString() should better
         // be executed outside synchronized blocks in order to reduce the
         // risks of causing deadlocks.
-        if (logger.isInfoEnabled())
-        {
-            logger.info("create_conf, id=" + conference.getID()
-                    + " gid=" + conference.getGid()
-                    + " logging=" + enableLogging);
-        }
+        logger.info(() -> "create_conf, id=" + conference.getID()
```

Comment:
I wonder if we should care about using suppliers for anything INFO and above. WARN is probably going to get logged, so the performance is the same. If we have enough WARN messages that are not being logged to affect performance we have another problem.

I think the convention so far has been to always require a level check for DEBUG and TRACE (even when it’s a single string), but never require it for INFO and above. But obviously here we're not using this convention. Is it worth all the extra code just to avoid either an `if`, or a few string concatenations when a conference is created?

Reply:
I phrased the above generally, but this is the only place in this PR where it applies.

Reply:
yeah, i wondered that too, but, for health checks, for example, we do set the level to warn (not info) so there is maybe some difference there (assuming we actually used this form everywhere, which i don't think we do).

Reply:
So why did you leave this in? It goes against the usual convention, it adds an extra method, it has to use a `final` local variable, and you seem to agree that its only use is for health checks, i.e. to avoid 6 string concatenations once every 10 second.

Reply:
And it was an unresolved comment on the PR.

Reply:
sorry, i didn't mean to steam roll this in or anything, but i didn't know you had an issue with the helper method (as you didn't mention that) and there was no response to my comment after multiple days.  i don't feel strongly about it so fine with changing it.